proposals/collection-expression-arguments.md

```diff
@@ -111,9 +111,9 @@ passed to that constructor, which is definitely is not.
 Based on all of the above, a small handful of options have come up that are felt to solve the needs of passing
 arguments, without stepping out of bounds of the goals of collection expressions.
 
-## Option 1: `[with(...arguments...)]`
+## `[with(...arguments...)]` Design
 
-The design of this form would be as follows:
+Syntax:
```

Comment:
Let's remove all KVPs from examples below, since not part of the feature

Reply:
There's still some Dictionary examples and KVP mentions that could use cleaning (line 169 below)

Reply:
Dictioanry is fine to mention (it's legal to construct witha  normal collection expression). I'll remove anything i can find that is kvp related.